lms/static/scripts/postmessage_json_rpc/server/server.js

```diff
@@ -61,22 +61,24 @@ export default class Server {
    * Receive a postMessage event and, if it's a JSON-RPC request from an
    * allowed origin, post back either a result or an error response.
    */
-  _receiveMessage(event) {
+  async _receiveMessage(event) {
     if (!this._allowedOrigins.includes(event.origin)) {
       return;
     }
 
     if (!this._isJSONRPCRequest(event)) {
       return;
     }
+
+    const result = await this._jsonRPCResponse(event.data);
+    event.source.postMessage(result, event.origin);
```

Comment:
I don't think it will make a practical difference, but was there a reason for moving this up from before the `this._resolveSidebarWindow` call?

Reply:
Good q. Yes, and no!

In short, I will move this back to the previous line to avoid confusion.  And to be clear, its the next bit of code that moved down which was the important part:

```
this._resolveSidebarWindow
```

_Backstory.._
At first I thought it fixed a timeout issue I saw after making these calls `async`... and in fact, it did, but it only masked the problem. Which makes more sense when I more closely understood was what going on in the client. We need to build out the `readyToReceive` RPC call that the client sends to LMS to only after its own RPC server is spun up. Once I build that out, this issue will be resolved. PR to come.